library/pod-security-policy/selinux/template.yaml

```diff
@@ -3,7 +3,7 @@ kind: ConstraintTemplate
 metadata:
   name: k8spspselinuxv2
   annotations:
-    description: Controls the SELinux context of the container.
+    description: "Defines an allow-list of seLinuxOptions configurations for pod containers.  Corresponds to a PodSecurityPolicy with SELinux rule `MustRunAs` enabled and specific `seLinuxOptions` specified."
```

Comment:
can we make these longer descriptions multi-lined via the `|` operator (like the rego field below). This should help readability for people using vim (or GitHub code review)

WRT text...

"and specific `seLinuxOptions` specified."

"specific" isn't very descriptive. Do you mean the options as provided to the parameters of the CT?

Reply:
Will change all of the templates to use the `|` operator.  It's definitely cleaner and more readable.

Reply:
r.e. "specific, the idea I'm going for is the following:

If you use this template, you configure at least one set of `seLinuxOptions` that is defined as a legal configuration of options for a container to run with.  Were you to apply a container with a different set of Linux Options, that container would be rejected by g8r.

To achieve the same effect with a PSP, you'd have to create an `seLinux` entry with rule `MustRunAs` and the specific set of allowed SELinux options included under it.  That would look like:

```yaml
spec:
  seLinux:
    rule: RunAsAny
    seLinuxOptions:
      [values]
```

So I'm really just trying to say that you'd have to configure that field with some specific values of your choosing.  So I'm not being particularly _specific_ when I say specific.  I'm just trying to say that you have to configure.

How could I say it better?

Reply:
One thing I forgot about is newlines... I think `>` is the correct character if you want to have newlines for readability but not for content:

https://yaml.org/spec/1.2.1/#style/block/folded

Reply:
re: specific... TBH I wouldn't mention it. It's enough to say that it is "requiring SELinux configs"

Referencing the PSP policy could be helpful for users already familiar with PSP, to help guide them when choosing replacement policies, but we shouldn't do it to the point where it confuses anyone unfamiliar with PSP and reading the docs for the first time.

Reply:
Are you saying I should get rid of the "Corresponds to a PodSecurityPolicy" parts of all the descriptions?  @mkorfgoogle told me he liked those.

Reply:
Also, newest version:

```
      Defines an allow-list of seLinuxOptions configurations for pod
      containers.  Corresponds to a PodSecurityPolicy requiring SELinux configs.
```